tests/playwright/sanity/modules/promotions/admin-menu-items/menu-item-promotions.test.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test';
+import WpAdminPage from '../../../../pages/wp-admin-page';
+
+test( 'Promotion screenshot', async ( { page }, testInfo ) => {
+	const wpAdminPage = new WpAdminPage( page, testInfo ),
+		promotionContainer = '.e-feature-promotion';
+
+	await wpAdminPage.login();
+	await page.goto( '/wp-admin/admin.php?page=e-form-submissions/' );
+	await page.waitForSelector( promotionContainer );
+	await expect( page.locator( promotionContainer ) ).toHaveScreenshot( 'promotion-menu-item-desktop.png' );
```

Comment:
Any particular reason this line was dropped from this Beta PR?
```await promoContainer.waitFor();```

Reply:
oh, yeah, I forgot it. But I think for beta we can skip it. The better code is already in main.

Reply:
It was redundant,
This will be fixed in https://elementor.atlassian.net/browse/ED-13391

Reply:
@Barcov 
in main we assigned an element to a constant. Then we wait for it, then do a screenshot. But it is one constant. Here we just duplication the code. We are waiting for the selector and then just get the same element with the selector to get a screenshot